README.md

```diff
@@ -1 +1,178 @@
-# toDoList
\ No newline at end of file
+![68747470733a2f2f696d672e796f75747562652e636f6d2f76692f416355642d5f596a6a71672f302e6a7067](https://user-images.githubusercontent.com/106747387/230112602-9404e3e1-7fbb-4690-98e3-bbffbf5c314a.jpeg)
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+    - [Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 To Do List <a name="about-project"></a>
+
+This is a list of tasks that an individual needs to complete or accomplish. Tasks are typically put in order by priority or importance. 
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>html</li>
+    <li>css</li>
+    <li>javascript</li>
+    <li>webpack</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- *Add what to do*
+- *Remove after completed*
+- *Great UI*
+- 
+<!-- Live Demo -->
+
+### Live Demo <a name="live-demo"></a>
+
+[visit here](https://oluwatobi-fasasi.github.io/toDoList/dist/)
+
```

Comment:
![image](https://github.com/oluwatobi-fasasi/toDoList/assets/93039359/4d3f260f-e93c-44bd-a046-2af9062da05f)

- [ ] It would be nice if you fix your Live Link because right now it is broken and giving me the error showing in the image above 👀 Kindly fix this link to make your readme descriptive and professional 👍